app/poros/food.rb

```diff
@@ -0,0 +1,15 @@
+class Food
+    attr_reader :hits,
+                :gtin_upc,
+                :description,
+                :brand_owner,
+                :ingredients
+
+    def initialize(attributes, total_hits)
+        @hits = total_hits
+        @gtin_upc = attributes[:gtinUpc]
```

Comment:
We can do checks in our PORO for nil values like such:
```
@gtin_upc = attributes[:gtinUpc] ? attributes[:gtinUpc] : 'No response'
```
This is called a ternary